src/components/connections.vue

```diff
@@ -46,4 +46,13 @@ export default {
 
 </template>
 
-<style scoped lang="scss"></style>
+<style scoped lang="scss">
+.collection {
+  cursor: pointer;
+  color: var(--list-hover-color);
+}
+.collection:hover {
+  color: var(--fg-color);
+  transition: var(--transition);
```

Comment:
You need to move `transition` to `.collection`, otherwise it won’t transition when you hover off.